_licenses/gfdl-1.3.txt

```diff
@@ -0,0 +1,476 @@
+---
+title: GNU Free Documentation License 1.3
+nickname: GNU FDL 1.3
```

Comment:
I would delete this. It's an optional field, and unlike GPLv3, there isn't a consistent nickname, even on the FSF site.